Ai Pin/Views/Notes/NotesEditView.swift

```diff
@@ -0,0 +1,63 @@
+import SwiftUI
+import OSLog
+
+struct NotesEditView: View {
+    @Environment(NavigationStore.self) private var navigationStore
+    @Environment(\.dismiss) private var dismiss
+    @State private var triggerSaveHatic = false
+    @State private var triggerCancelHatic = false
```

Comment:
these don't appear to be doing anything.

Reply:
They were in fact not doing anything 🤦🏻‍♂️. I removed the cancelled haptic and just kept the save haptic to provide some feedback.